src/map/skill.cpp

```diff
@@ -9997,13 +10000,11 @@ int skill_castend_nodamage_id (struct block_list *src, struct block_list *bl, ui
 		break;
 
 	case NPC_POWERUP:
-		sc_start(src,bl,SC_INCATKRATE,100,200,skill_get_time(skill_id, skill_lv));
 		clif_skill_nodamage(src,bl,skill_id,skill_lv,
 			sc_start(src,bl,type,100,100,skill_get_time(skill_id, skill_lv)));
```

Comment:
Val1 changes from 200 -> 100 and is not used in general.

Reply:
Value not needed. The increase it coded into battle.cpp.